2022-03-07-report.org

```diff
@@ -0,0 +1,150 @@
+#+title:  Weekly Report for 28th Feb to 7th March 2022
+#+author: Philip Herron
+#+date:   2022-03-07
+
+** Overview
+
+Thanks again to [[https://opensrcsec.com/][Open Source Security, inc]] and [[https://www.embecosm.com/][Embecosm]] for their ongoing support for this project.
+
+*** Milestone Progress
+
+We are in the final few weeks to close out declarative macros by the end of the month; this should be the last prominent feature required before compiling more of an older version of libcore. We should be roughly on time, maybe a few days out, but nothing significant stands out as a roadblock right now. I imagine we might move some of the built-in macros to be part of the builtins and intrinsics ongoing milestone since some of these may require more builtins such as panic!.
+
+To keep on track with our aggressive timeline, we want to begin investigations into achieving our crate metadata exports with GCC since we have several options available to us, such as LTO streaming, CPP modules, or roll our own system such as GCCGO's .go_export section in object files with a simplified AST.
+
+**** Monthly Community Call
+
+We had our regular community call on 4th March 2022, please find our meeting notes over on: https://github.com/Rust-GCC/Reporting/blob/main/2022-03-04-community-call.md
+
+** Completed Activities
+
+- Refactor substitution context during macro expansion to be in its own file [[https://github.com/Rust-GCC/gccrs/pull/981][PR981]]
+- Enforce quites during command line cfg arguments [[https://github.com/Rust-GCC/gccrs/pull/983][PR983]]
+- Bugfix memory corruption of lexing string buffers [[https://github.com/Rust-GCC/gccrs/pull/988][PR988]]
+- Remove bad lambda from iteration of arguments on function types [[https://github.com/Rust-GCC/gccrs/pull/984][PR984]]
+- Add must_use attribute support [[https://github.com/Rust-GCC/gccrs/pull/990][PR990]]
+- Bug fix parsing macro invocation with semicolon's during statement contexts [[https://github.com/Rust-GCC/gccrs/pull/985][PR985]]
+- Fix ICE during recursive macro invocations [[https://github.com/Rust-GCC/gccrs/pull/986][PR986]]
+- Support match seperators '|' in macro rules [[https://github.com/Rust-GCC/gccrs/pull/991][PR991]]
```

Comment:
```suggestion
- Support repetition separators in macro rules [[https://github.com/Rust-GCC/gccrs/pull/991][PR991]]
```
I used `|` as an exemple but it works for everything now, whereas it didn't work for anything before